src/mwoffliner.lib.ts

```diff
@@ -612,7 +612,7 @@ async function execute(argv: any) {
     }
 
     function createMainPageRedirect() {
-      logger.log(`Create main page redirection from [index] to[${'A/' + dump.getArticleBase(mainPage, true)}]`);
+      logger.log(`Create main page redirection from [index] to[${'A/' + encodeURIComponent(mainPage)}]`);
```

Comment:
Same here? I don't understand why the URL in the ZIM should have any encoding.

Reply:
Removed.
Anyways it's just in log, seems sense to remove this.